tests/behavioural/features/steps/test_send_message.py

```diff
@@ -30,6 +34,42 @@ def step_impl_visit_create_message_page(context):
 def step_impl_add_text_to_subject_and_body(context):
     context.browser.find_by_id('secure-message-subject').send_keys('Test Subject')
     context.browser.find_by_id('secure-message-body').send_keys('Test Body')
+
+
+@when('I have a message with non alpha characters')
+def step_impl_add_non_alpha_text_to_subject_and_body(context):
+    context.browser.find_by_id('secure-message-subject').send_keys('[]!@£$%^&*()')
+    context.browser.find_by_id('secure-message-body').send_keys('"?><:|}{+_±')
```

Comment:
This test wont pass without extra environment variables.
We should either specify these somewhere or reject the characters that aren't acceptable without them

Reply:
So with this I think that we need to know what characters we'd actually need to include.

Reply:
good point, I'm not sure that we'd be required to support accepting `±` for example